vNext/src/Orleans/Shims/ClassesForTesting/CustomSerializerTestClasses.cs

```diff
@@ -0,0 +1,116 @@
+﻿using System;
+using Orleans.CodeGeneration;
+using Orleans.Serialization;
+
+namespace UnitTests.CustomSerializerTestClasses
```

Comment:
what's this for? seems strange to have a shim that is only used for testing? why is this not part of the testing code?